lib/docker.ts

```diff
@@ -89,6 +89,33 @@ class Docker {
     ]);
   }
 
+  public async pull() {
+    return subProcess.execute("docker", ["pull", this.targetImage]);
+  }
+
+  public async isImageLocal(targetImage: string) {
+    try {
+      await this.inspectImage(targetImage);
+      return true;
+    } catch (err) {
+      return false;
+    }
+  }
+
+  public async inspectWithPull(targetImage: string) {
+    const isImageLocal = await this.isImageLocal(targetImage);
+    if (!isImageLocal) {
+      try {
+        await this.pull();
```

Comment:
The called function has no parameters and only a single line of code - not sure we gain anything here. Is there a reason why the single line of code should be in a function?

Reply:
... but thinking about it, it would make sense to move the surrounding try catch block into the function.

Reply:
The idea was to keep it the same as the rest of the functions in Docker, in terms of returned type etc.
Do you think it's not worth it?